doc/source/api.rst

```diff
@@ -19,6 +19,22 @@ and methods.
    api/variables
 
 
+**Notebook logging behaviour**
+
+Pyam wants to provide sensitive defaults for users unfamiliar with
+`setting up python's logging library <https://realpython.com/python-logging/#basic-configurations>`_,
+and therefore will provide a basic configuration by invoking
+
+.. code-block:: python
+
+   import logging
+   logging.basicConfig(level="INFO", format="%(name)s - %(levelname)s: %(message)s")
+
+if (and only if):
+
+1. it determines to be running within a notebook, and
+2. logging is still *unconfigured by the time the first logging message is to be emitted*.
```

Comment:
this is "the first logging message to be emitted **by pyam**", right? Or really any logging message?

Reply:
Indeed